tests/scripts/github-action-helper.sh

```diff
@@ -562,7 +562,9 @@ function get_secret_key() {
 }
 
 function s3cmd() {
-  command timeout 150 s3cmd -v --config=s3cfg --access_key="${S3CMD_ACCESS_KEY}" --secret_key="${S3CMD_SECRET_KEY}" "$@"
+ command timeout 200 bash <<EOF
```

Comment:
Spacing got messed up here slightly. Let's make sure it's 2-space indented on next iterations

```suggestion
  command timeout 200 bash <<EOF
```